modules/ovirt-csi-driver-storage-class.adoc

```diff
@@ -1,30 +1,42 @@
 // Module included in the following assemblies:
 //
 // * storage/container_storage_interface/persistent-storage-csi-ovirt.adoc
+// * post_installation_configuration/storage-configuration.adoc
 
 [id="ovirt-csi-driver-storage-class_{context}"]
-= oVirt CSI driver storage class
+ifeval::["{context}" == "post-install-storage-configuration"]
+= {rh-virtualization-first} object definition
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-ovirt"]
+= {rh-virtualization-first} CSI driver storage class
+endif::[]
+
 
 {product-title} creates a default object of type `StorageClass` named `ovirt-csi-sc` which is used for creating dynamically provisioned persistent volumes.
 
 To create additional storage classes for different configurations, create and save a file with the `StorageClass` object described by the following sample YAML:
 
+.ovirt-storageclass.yaml
 [source,yaml]
 ----
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: <storage-class-name>  <1>
   annotations:
-    storageclass.kubernetes.io/is-default-class: "false"  <2>
+    storageclass.kubernetes.io/is-default-class: "<boolean>"  <2>
 provisioner: csi.ovirt.org
+allowVolumeExpansion: <boolean> <3>
+reclaimPolicy: Delete
```

Comment:
reclaimPolicy - Dynamically provisioned PersistentVolumes of this storage class are created with this reclaimPolicy. Defaults to Delete.

Reply:
@Gal-Zaidman @mgold1234 What are the other values? What does `Delete` do?

Reply:
@stoobie , I think you can add explanation to reclaimPolicy: Delete
volumeBindingMode: Immediate. VolumeBindingMode -  indicates how PersistentVolumeClaims should be provisioned and bound. When unset, VolumeBindingImmediate is used. This field is only honored by servers that enable the VolumeScheduling feature.

Reply:
Done.